### ISSUE TITLE
open statements lead to react error

### ISSUE THREAD

**Issue by whitetigle** [fable-compiler/fable-react#95]
Hi,
just stumbled one hour on this problem: 

this does not work
```f#
  open Fable.Recharts
  open Fable.Recharts.Props
  module SVG = Fable.Helpers.React.Props
  open Fable.Helpers.React
```
and leads to this error: 

> warning.js:33 Warning: React does not recognize the `dataKey` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `datakey` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

meanwhile this works 
```f#
  open Fable.Recharts
  open Fable.Recharts.Props
  module SVG = Fable.Helpers.React.Props
  // open Fable.Helpers.React
```

maybe something with modules loading?

**Comment by whitetigle** [comment]
BTW: `Fable 1.3.17`

**Comment by alfonsogarciacaro** [comment]
That's weird, Fable doesn't do anything for opening statements. Maybe the last opening is shadowing something from Recharts, does this work?

```fsharp
  open Fable.Helpers.React
  open Fable.Recharts
  open Fable.Recharts.Props
  module SVG = Fable.Helpers.React.Props
```

**Comment by MangelMaxime** [comment]
Seems like `DataKey` is defined only in recharts bindings. Perhaps you used it on an HTML element ?

**Comment by swrhim** [comment]
I ran into the same problem where ordering of the open statements was preventing my chart for recharts from being rendered with the same error message. Should this issue be referenced from this [repo](https://github.com/fable-compiler/fable-recharts)?

**Comment by alfonsogarciacaro** [comment]
Yes, that may be a good idea. Maybe opening an issue in fable-rcharts with a descriptive title that references this one. What Fable version are you using btw?

**Comment by swrhim** [comment]
Version 3. I'm using the latest SAFE template before .net 5 was announced. (not sure if that has something different)
